kimchi/js/bindings.js

```diff
@@ -1565,33 +1565,26 @@ var caml_plonk_verification_evals_to_rust = function (
   x,
   klass,
   poly_comm_class,
-  mk_affine
+  mk_affine,
```

Comment:
I think we don't need either of mk_affine and `poly_comm_class` once we have `poly_comm_to_rust`

Reply:
Good catch! Fixed.